rotkehlchen/globaldb/handler.py

```diff
@@ -258,16 +259,17 @@ def retrieve_assets(filter_query: 'AssetsFilterQuery') -> Tuple[List[Dict[str, A
         """  # noqa: E501
         query = f'SELECT * FROM ({parent_query}) {prepared_filter_query}'
         underlying_tokens_query = """
-        SELECT assets.identifier AS identifier, address, token_kind, parent_token_entry, weight, name, symbol, type, chain
+        SELECT parent_token_entry AS identifier, underlying_tokens_list.identifier, token_kind, weight, name, symbol, type, chain
```

Comment:
Are `name, symbol, type, chain` used anywhere?

Reply:
they were needed due the filter